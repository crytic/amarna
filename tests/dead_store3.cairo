func are_equal(x, y) -> (eq):
    let (a) = 44
    if x == y:
        return (FALSE)
    end
    if x == a:
        return (TRUE)
    end
end
