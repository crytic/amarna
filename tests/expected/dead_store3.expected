[unused-function] in dead_store3.cairo:1:1